roles/container-engine/containerd/templates/containerd.service.j2

```diff
@@ -30,7 +30,6 @@ RestartSec=5
 # in the kernel. We recommend using cgroups to do container-local accounting.
 LimitNPROC={{ containerd_limit_proc_num }}
 LimitCORE={{ containerd_limit_core }}
-LimitNOFILE={{ containerd_limit_open_file_num }}
 LimitMEMLOCK={{ containerd_limit_mem_lock }}
```

Comment:
This is also not in containerd upstream service file, right ?

Reply:
We are matching If you check it here: https://github.com/containerd/containerd/blob/main/containerd.service#L30-L33

Reply:
I don't see LimitMEMLOCK here ^

Reply:
Yeah and that was done previously here for a reason: https://github.com/kubernetes-sigs/kubespray/pull/9269

Reply:
Urgh. The reason if I follow the issue is basically: add that particular setting with no motivation :disappointed: 
I guess it's fine for now, but franky, we would better off in the future shipping a pristine service and let users provide a full systemd override file so users can tweak whatever they like without requesting specific systemd settings.